backend/api/Services/RobotService.cs

```diff
@@ -127,6 +134,15 @@ public async Task<Robot> Update(Robot robot)
             return robot;
         }
 
+        public async Task<IList<Robot>> ReadLocalizedRobotsForInstallation(string installationCode)
```

Comment:
This looks good, but if we keep adding more service functions like these then we could perhaps make a generic filter function like we did for missionRun and missionDefinition

Reply:
It might be relevant although the filter functions for mission run and mission definition are more linked to there being a lot of them and there is a need to be able to search specifically (for example by date etc.). As long as there are relatively few read functions for robots I think it's fine.

In my opinion this is also a lot more readable than having one function which we send a filter into. Easier to have a good, readable function name this way.